plenary/plenary-25.yaml

```diff
@@ -0,0 +1,340 @@
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Confirmation of ISO 8601:2004"
+    identifier: 287
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154, having considered the result of the systematic review of ISO 8601:2004 Data elements and interchange formats – Information interchange Representation of dates and times (document 154n507), resolves to confirm ISO 8601:2004
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Confirmation of ISO/TS 15000: 2004 parts 1 through 4"
+    identifier: 288
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+
+    actions:
+      - type: resolves
+        date_effective:
+          start: 2008-11-25
+          end: 2008-11-26
+        message: ISO/TC 154, having considered the result of the systematic review of ISO/TS 15000:2004 Electronic business eXtensible Markup Language (ebXML) parts 1 through 4 (document 154n508 – 154n511), resolves to confirm ISO/TS 15000: 2004 parts 1 through 4
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Resignation of Mr. Vuilleumier as chair and secretary of ISO/TC 154"
+    identifier: 289
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: ACCLAMATION
+
+    actions:
+      - type: thanks
+        degree: unanimous
+        message: ISO/TC 154 notes with regret the resignation of Mr. Vuilleumier as chair and secretary of ISO/TC 154 and thanks him for the excellent and important work he has done in the area of standards for trade facilitation. His enthusiasm and leadership of ISO/TC 154 have played an important role in the achievements of ISO/TC 154
+
+
+  - dates:
+      - 2008-11-25
+      - 2008-11-26
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appreciation"
+    identifier: 290
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: ACCLAMATION
+
+    actions:
+      - type: thanks
+        degree: unanimous
+        message: ISO/TC 154 expresses its sincere appreciation to the World Customs Organisation for arranging the twenty-fifth meeting of ISO/TC 154 in Brussels and hospitality
```

Comment:
I'm not sure about the setting of these two sections. 
I used the word `ACCLAMATION` as message of the approval, and `thanks` as type of action. Is that correct?

Original source reference:
![capture2](https://user-images.githubusercontent.com/33627611/71758744-fedc6a80-2e79-11ea-9fc4-df836c01d243.JPG)

Reply:
Agreed!